msteams-platform/teams-faq.md

```diff
@@ -1212,10 +1212,7 @@ Before provision, the toolkit asks you if you want to create a new resource grou
 <details>
 <summary>How can I provision a SharePoint-based app?</summary>
 
-You can follow [provision SharePoint-based app](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&tutorial-step=4).
-
-> [!NOTE]
-> Building a Teams app using SharePoint Framework with Teams Toolkit doesn't have direct integration with Microsoft Entra admin center. The content in the document doesn't apply to SPFx-based apps.
+You can do so by following [provision SharePoint-based app](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&tutorial-step=4). Building a Teams app using SharePoint Framework with Teams Toolkit does not have direct integration with the Microsoft Entra admin center.
```

Comment:
Please update:

Building a Teams app using SharePoint Framework with Teams Toolkit does not have direct integration with the Microsoft Entra admin center. To provision a SharePoint-based app, follow the [provision SharePoint-based app](/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&tutorial-step=4) guide.